### DISCUSSION TITLE
Error pushing to GitHub repository: Repository not found

### DISCUSSION THREAD

**Discussion by sadok-dorbez** [community/community#50073] [Actions] [answered]
### Select Topic Area

Question

### Body

Hello everyone,

I'm having an issue pushing to a GitHub repository. Every time I try to push my changes using the command "git push -u origin [branch-name]", I get an error message that says "remote: Repository not found. fatal: repository '[repository URL]' not found".

I have checked and double-checked that I am a collaborator and have the necessary access rights to the repository. I have also made sure that the repository URL is correct and that I have spelled everything correctly. However, I still cannot push my changes.

I have tried creating a new branch and pushing to that branch, as well as creating a new repository and trying to push my changes to that repository. None of these attempts have worked, and I keep getting the same error message.

I have also checked my firewall settings and temporarily disabled my firewall, but this did not solve the issue either.

I would greatly appreciate any suggestions or advice on how to resolve this issue. Has anyone else encountered a similar issue when pushing to a GitHub repository? Any help would be greatly appreciated.

Thank you in advance for your help.

**Comment by maahaMaya** [comment, marked as answer]
**git push origin master --force** 
use these command 
It will work

**Comment by sadok-dorbez** [comment]
Probelm Solved
I use Git on Windows, I tried to clear my credentials:

Locate "credential manager" 
Remove all credentials related to GitHub
